features/step_definitions/portfolio.rb

```diff
@@ -1,4 +1,4 @@
-require 'capybara/cucumber'
+﻿require 'capybara/cucumber'
```

Comment:
> Each committed line is (...) useful (it would not work if removed)

Pas de différence claire avec la ligne modifiée (et encore moins en relation avec la livraison).